app/models/concerns/startdate_beyond_osra_establishment_validation.rb

```diff
@@ -0,0 +1,12 @@
+module StartdateBeyondOsraEstablishmentValidation
+	include ValidatorHelpers
+
+	OSRA_ESTABLISHMENT_DATE = Date.new(2012,04,01)
```

Comment:
yes, we were thinking about it. There is also the priority countries for the select option, the pagination page limit, etc
I would like to have them all in one file.